index.js

```diff
@@ -71,9 +71,31 @@ var allTargets = [
   {runtime: 'electron', target: '1.0.2', abi: '47', lts: false},
   {runtime: 'electron', target: '1.2.8', abi: '48', lts: false},
   {runtime: 'electron', target: '1.3.13', abi: '49', lts: false},
-  {runtime: 'electron', target: '1.4.15', abi: '50', lts: false}
+  {runtime: 'electron', target: '1.4.15', abi: '50', lts: false},
+  {runtime: 'electron', target: '1.5.0', abi: '51', lts: false},
+  {runtime: 'electron', target: '1.6.0', abi: '53', lts: false}
 ]
 
+var deprecatedTargets = [
+  {runtime: 'node', target: '0.2.0', abi: '1', lts: false},
+  {runtime: 'node', target: '0.9.1', abi: '0x000A', lts: false},
+  {runtime: 'node', target: '0.10.0', abi: '0x000B', lts: false},
+  {runtime: 'node', target: '0.11.0', abi: '0x000C', lts: false},
+  {runtime: 'node', target: '0.11.10', abi: '13', lts: false},
+  {runtime: 'node', target: '1.0.0', abi: '42', lts: false},
+  {runtime: 'node', target: '1.1.0', abi: '43', lts: false},
+  {runtime: 'node', target: '2.0.0', abi: '44', lts: false},
+  {runtime: 'node', target: '3.0.0', abi: '45', lts: false},
+  {runtime: 'node', target: '8.0.0', abi: '52', lts: false},
+  {runtime: 'electron', target: '0.30.0', abi: '44', lts: false},
+  {runtime: 'electron', target: '0.31.0', abi: '45', lts: false},
+  {runtime: 'electron', target: '0.33.0', abi: '46', lts: false}
```

Comment:
what difference does it make?

i made it the first one supporting it because that makes it easy to write a generic `getAbi` function that only uses this array as input